devicetypes/smartthings/hue-bridge.src/hue-bridge.groovy

```diff
@@ -15,19 +15,27 @@ metadata {
 		// TODO: define status and reply messages here
 	}
 
-	tiles {
+	tiles(scale: 2) {
+     	multiAttributeTile(name:"rich-control"){
+			tileAttribute ("", key: "PRIMARY_CONTROL") {
```

Comment:
I did test and it worked :) but haven't tested in older clients. Should I ask QA specifically about this? I don't have older clients